src/manifold/src/quickhull.h

```diff
@@ -264,86 +242,86 @@ class MeshBuilder {
 
     // Create halfedges
     HalfEdge AB;
-    AB.endVertex = b;
-    AB.opp = 6;
-    AB.face = 0;
+    AB.halfEdgeManifold.endVert = b;
```

Comment:
Didn't we talk about condensing this huge list?

Reply:
Yes, sorry I forgot about that will change this.